### ISSUE TITLE
Accuracy Not Displayed in App

### ISSUE THREAD

**Issue by aswinkumar1999** [CleanPegasus/TensorDash#8]
Saw this in Reddit and thought it was cool, so wanted to give it a try, The Loss function was correct but the accuracy wasn't displayed.

[Colab notebook](https://colab.research.google.com/drive/1o_-d_vbRIpNBq1F9ljj88GLnNvPvROO-) with which I tested this : 

```https://colab.research.google.com/drive/1o_-d_vbRIpNBq1F9ljj88GLnNvPvROO-```

Attaching a Picture of the App 

![Screenshot_20200218-234632](https://user-images.githubusercontent.com/19380153/74765522-c1a52e00-52a9-11ea-8d89-95a6946c50df.jpg)

**Comment by CleanPegasus** [comment]
Thanks for bringing it to our concern. The package was made for tensorflow versions greater than 2.0. But colab uses tensorflow 1.14 by default.
We fixed the it to work with tensorflow versions lesser than 2.0. Please update the tensor-dash package to version 1.4.1.

**Comment by CleanPegasus** [comment]
*Kindly update to tensor-dash version 1.4.2
